### PR TITLE
[improve][doc] Update the development environment to build the software version

### PR DESCRIPTION
**Pull request by SnowMoon-Dev**
…re version

## Purpose of the pull request

Update the development environment to build the software version

## Verify this pull request

Manually verified the change by testing locally.

### CODE REVIEW

File: docs/docs/en/development/development-environment-setup.md [hunk 1]
```diff
@@ -2,12 +2,11 @@
 
 ## Software Requests
 
-Before setting up the DolphinScheduler development environment, please make sure you have installed the software as below:
```
Comment (SbloodyS) [inline docs/docs/en/development/development-environment-setup.md:5]: I think this description needs to be retained.

File: docs/docs/zh/development/development-environment-setup.md [hunk 2]
```diff
@@ -1,13 +1,12 @@
 # DolphinScheduler 开发手册
 
-## 前置条件
+## 软件要求
 
-在搭建 DolphinScheduler 开发环境之前请确保你已经安装一下软件
```
Comment (SbloodyS) [inline docs/docs/zh/development/development-environment-setup.md:5]: So as english's doc.

File: docs/docs/en/development/development-environment-setup.md (hunk 1 above)
Comment (SnowMoon-Dev) [reply docs/docs/en/development/development-environment-setup.md:5]: ok，plus it's more user-friendly

**Review by SbloodyS — APPROVED** [review]
LGTM. @zhongjiajie Do you have better suggestions?

**Comment by zhongjiajie** [comment]
Thanks

**Pull request merged**